src/create-draggable.ts

```diff
@@ -78,7 +78,7 @@ const createDraggable = (id: Id, data: Record<string, any> = {}): Draggable => {
 
           if (!transformsAreEqual(resolvedTransform, noopTransform())) {
             const style = transformStyle(transform());
-            element.style.setProperty("transform", style.transform);
+            element.style.setProperty("transform", style.transform ?? null);
```

Comment:
these were causing ts errors in build (`undefined is not assignable to string | null`)